### DISCUSSION TITLE
Questions concernant le menu

### DISCUSSION THREAD

**Discussion by Jad31** [TempoDev/Arcade-Toulouse#16] [Questions]
Bonjour tout le monde,

Le menu est-il une scène ?

Et si oui, comment comptez-vous l'implémenter ?

**Comment by emneo-dev** [comment]
- oui
- jsp

**Comment by nasso** [comment]
Je pense que ça va pas mal dépendre de ton archi de ton côté, mais perso je le vois plus comme un jeu "built-in" dans le core qui va utiliser la lib graphique pour l'affichage, mais va être bien plus intégré au core afin de pouvoir faire des actions comme charger un jeu par exemple.
